src/Xamarin.Android.Build.Tasks/Microsoft.NET.Sdk.Android/WorkloadManifest.in.json

```diff
@@ -69,6 +69,22 @@
       "kind": "framework",
       "version": "@WORKLOAD_VERSION@"
     },
+    "Microsoft.Android.Runtime.NativeAOT.35.android-arm": {
+      "kind": "framework",
+      "version": "@WORKLOAD_VERSION@"
+    },
+    "Microsoft.Android.Runtime.NativeAOT.35.android-arm64": {
+      "kind": "framework",
+      "version": "@WORKLOAD_VERSION@"
+    },
+    "Microsoft.Android.Runtime.NativeAOT.35.android-x86": {
+      "kind": "framework",
+      "version": "@WORKLOAD_VERSION@"
+    },
+    "Microsoft.Android.Runtime.NativeAOT.35.android-x64": {
+      "kind": "framework",
+      "version": "@WORKLOAD_VERSION@"
+    },
```

Comment:
As we continue to expand this list, we probably will want to revisit https://github.com/dotnet/sdk/issues/24077 / https://github.com/dotnet/sdk/pull/24873. For .NET 10, we're very much going to want to come up with a way to cut down on the (now 8, ~soon to be 12?) duplicate versions of Mono.Android.dll (~35 MB each) and friends that we will be shipping.